src/main/java/com/meta/cp4m/message/FBMessageHandler.java

```diff
@@ -44,6 +44,7 @@ public class FBMessageHandler implements MessageHandler<FBMessage> {
   private final String appSecret;
 
   private final String accessToken;
+  private final Boolean isInstagram;
```

Comment:
in general you want to use primitives `bool` where possible because it is not possible for them to be `null`